lib/player.rb

```diff
@@ -0,0 +1,29 @@
+# rubocop:disable Lint/UnreachableCode
+
+class Player
+  attr_accessor :token, :name
+  def initialize(name, token)
+    @name = name
+    @token = token
+  end
+
+  def to_s
+    "Player: #{@name}--#{@token}"
+  end
+
+  def get_choice(arg)
+    loop do
+      puts 'Please enter a number between 1 to 9: '
+      choice = gets.chomp.to_i
+      index = choice - 1
+      if arg.include? choice
+        return index
+        break
+      else
+        puts 'Wrong position, please choose another one'
+      end
```

Comment:
- [ ] You shouldn't use `puts/print` or `gets.chomp` statements outside of the `bin/main.rb` file. 
screenshot of the requirements 👇 

![Screen Shot 2020-08-28 at 1 08 02 AM](https://user-images.githubusercontent.com/48804258/91503505-ed454880-e8ca-11ea-97d3-87e1a34c0c3c.png)

Reply:
:100: